app/blueprints/fund_builder/templates/round.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base.html" %}
+{% from "macros/wtfToGovUk.html" import input %}
+{% from "macros/wtfToGovUk.html" import yes_no %}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{%- from "govuk_frontend_jinja/components/select/macro.html" import govukSelect -%}
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <h1 class="govuk-heading-l">Create a Round</h1>
+        <div class="govuk-form-group">
+            <fieldset class="govuk-fieldset">
+                <form method="POST">
+                    {{ form.hidden_tag()}}
+                    {{ govukSelect({
+                        "id": form.fund_id.id,
+                        "name": form.fund_id.name,
+                        "label": {
+                          "text": form.fund_id.label
+                        },
+                        "items": all_funds})}}
+                    {{input(form.title_en)}}
+                    {{input(form.short_name)}}
+                    {{input(form.opens)}}
+                    {{input(form.deadline)}}
+                    {{input(form.assessment_start)}}
+                    {{input(form.reminder_date)}}
+                    {{input(form.assessment_deadline)}}
```

Comment:
not sure how finalised this one is to be but hints here for date format would be useful (or even a datetime field input:https://design-system.service.gov.uk/components/date-input/ )

Reply:
Yes totally. Very much just a dev-usage page atm. Didn't spend too much time on UI until mika has designed the proper journey